ArjunaJTA/recovery/README.md

```diff
@@ -1,99 +1,103 @@
-JBoss, Home of Professional Open Source Copyright 2008, Red Hat Middleware 
```

Comment:
removed copyright?

Reply:
the copyright was left as it was just it's displayed as the comment